rego/rego.go

```diff
@@ -14,6 +14,8 @@ import (
 	"strings"
 	"time"
 
+	"golang.org/x/exp/maps"
```

Comment:
I'm expecting this to break the compat builds because the package expects golang 1.20. But let's see for ourselves 🤞

Reply:
Didn't break. OK cool.

Reply:
On second thoughts, let's not get an exp package into the main code path. I'll update.